src/lib/chips/chip-list.spec.ts

```diff
@@ -353,9 +353,9 @@ describe('MatChipList', () => {
 
     });
 
-    it('should not float placeholder if no chip is selected', () => {
+    it('should float placeholder if chip is selected', () => {
```

Comment:
I think this was testing when there's a chip present but not selected, so it should still be `"... if no chip is selected"`. 

@tinayuangao Why was this expected to be not floating in the first place? It seems like we would want it to float even if the chip isn't selected, to avoid overlap.

Reply:
This test was invalid, it was expecting the inverse logic that was the bug before.

Reply:
agreed, but it was the "not float" part that was invalid not the "no chip is selected" part